fields/types/gcsimage/GcsImageField.js

```diff
@@ -1 +1,203 @@
-module.exports = require('../cloudinaryimage/CloudinaryImageField');
+import Field from '../Field';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import classnames from 'classnames';
+import get from 'lodash/get';
+import { Button, FormField, FormInput, FormNote } from 'elemental';
+
+const _ = {
+  get,
+};
+
+const SUPPORTED_TYPES = ['image/gif', 'image/png', 'image/jpeg', 'image/bmp', 'image/x-icon', 'application/pdf', 'image/x-tiff', 'image/x-tiff', 'application/postscript', 'image/vnd.adobe.photoshop', 'image/svg+xml'];
```

Comment:
`image/tiff` duplicate?

Reply:
Besides, do we need to confirm the supported image formats rather than extend? Skimming through some of the image manipulating packages, it seemed to me that not all the formats are not supported and even I don't think `graphicmagick` could extract the iptc from all of them.

Reply:
Yeah, we could confirm support types with photographers.
In this PR, if you feel ok, then I would like to change support types to 
`image/gif`, `image/png`, `image/jpeg` and `image/svg+xml`.

Reply:
This is graphicsmagick support formats list.
http://www.graphicsmagick.org/formats.html

Reply:
> Yeah, we could confirm support types with photographers.
> In this PR, if you feel ok, then I would like to change support types to
> `image/gif`, `image/png`, `image/jpeg` and `image/svg+xml`.
> 

`image/tiff` is also common format for the state of the art image.

> If you agree, I would like to finish this PR first.
> The support format confirmation could be in another thread.

Sure.